### DISCUSSION TITLE
Access model in setupShowOperation()

### DISCUSSION THREAD

**Discussion by colincameron** [Laravel-Backpack/community-forum#609] [Q&A (Help)] [answered]
Is it possible to access the model in setupShowOperation()?

Basically I want to be able to conditionally add a column based on a property of the model.

e.g.

```
protected function setupShowOperation()
{
    CRUD::column(...);
    /* ... */

    if ($model->is_manager) { // <-- I want to be able to do this
        CRUD::column(...);
    }
}
```

**Comment by karandatwani92** [comment, marked as answer]
Hi @colincameron ... Yes, you can access the model inside setupShowOperation via `$this->crud->getCurrentEntry();` 

checkout more available methods here https://backpackforlaravel.com/docs/6.x/crud-api#getters

**Reply by colincameron** [reply]
That's perfect thank you!
